src/String/Extra.elm

```diff
@@ -959,3 +960,40 @@ nonEmpty string =
         Nothing
     else
         Just string
+
+
+{-| Remove accents from string.
+
+    removeAccents "andré" == "andre"
+    removeAccents "Atenção" == "Atencao"
+-}
+removeAccents : String -> String
+removeAccents string =
+    if String.isEmpty string then
+        string
+    else
+        let
+            matches =
+                [ ( "[à-æ]", "a" )
+                , ( "[À-Æ]", "A" )
+                , ( "ç", "c" )
+                , ( "Ç", "C" )
+                , ( "[è-ë]", "e" )
+                , ( "[È-Ë]", "E" )
+                , ( "[ì-ï]", "i" )
+                , ( "[Ì-Ï]", "I" )
+                , ( "ñ", "n" )
+                , ( "Ñ", "N" )
+                , ( "[ò-ö]", "o" )
+                , ( "[Ò-Ö]", "O" )
+                , ( "[ù-ü]", "u" )
+                , ( "[Ù-Ü]", "U" )
+                , ( "ý", "y" )
+                , ( "ÿ", "y" )
+                , ( "Ý", "Y" )
+                ]
```

Comment:
Could you move this to a top level function? The ideally we would build the `Regex` only once, and not once per each call to the function:

```elm
accentRegex : List Regex
accentRegex = 
    ...
```